pyccel/parser/semantic.py

```diff
@@ -2274,17 +2274,18 @@ def _visit_Return(self, expr, **settings):
 
     def _visit_FunctionDef(self, expr, **settings):
 
-        name         = str(expr.name)
-        name         = name.replace("'", '')
-        cls_name     = expr.cls_name
-        hide         = False
-        kind         = 'function'
-        decorators   = expr.decorators
-        funcs        = []
-        sub_funcs    = []
-        is_pure      = expr.is_pure
-        is_elemental = expr.is_elemental
-        is_private   = expr.is_private
+        name            = str(expr.name)
+        name            = name.replace("'", '')
+        cls_name        = expr.cls_name
+        hide            = False
+        kind            = 'function'
+        decorators      = expr.decorators
+        funcs           = []
+        sub_funcs       = []
+        func_interfaces  = []
+        is_pure         = expr.is_pure
+        is_elemental    = expr.is_elemental
+        is_private      = expr.is_private
```

Comment:
I assume that you changed all these lines to align the `=` signs after you added one line:
```python
func_interfaces = []
```
But the result is not exactly what you intended, because you have an extra space just before the equal sign!

Reply:
yea, I didn't notice it